packages/ui/feature-builder-form-controls/src/lib/components/connections-dropdown-control/connections-dropdown-control.component.ts

```diff
@@ -0,0 +1,123 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import {
+  PieceConnectionDropdownItem,
+  PieceMetadataModel,
+  UiCommonModule,
+} from '@activepieces/ui/common';
+import { FormControl } from '@angular/forms';
+import { PieceAuthProperty } from '@activepieces/pieces-framework';
+import { DropdownSelectedValuesPipe } from '../../pipes/dropdown-selected-values.pipe';
+import { UiFeatureConnectionsModule } from '@activepieces/ui/feature-connections';
+
+//TODO: check add-edit-connection-button and remove unnecessary properties and ask mo about triggerName
```

Comment:
What's up?

Reply:
so we pass triggerName to connections-button to check if the trigger is an app webhook, if it is we don't show "Use your own credentials" but in some places outside the builder "connections table", it will show that, is it okay ?

Reply:
Fine for now